gitlab/v4/objects/merge_requests.py

```diff
@@ -327,6 +327,24 @@ def rebase(self, **kwargs: Any) -> Union[Dict[str, Any], requests.Response]:
         data: Dict[str, Any] = {}
         return self.manager.gitlab.http_put(path, post_data=data, **kwargs)
 
+    @cli.register_custom_action("ProjectMergeRequest")
+    @exc.on_http_error(exc.GitlabMRResetApprovalError)
+    def reset_approvals(
+        self, **kwargs: Any
+    ) -> Union[Dict[str, Any], requests.Response]:
+        """Attempt to reset approvals onto the target Merge Request
```

Comment:
Maybe we can use the upstream wording, `attempt` was more for rebase I'd say:
```suggestion
        """Clear all approvals of the merge request.
```